fluence-faas/src/faas.rs

```diff
@@ -29,10 +29,51 @@ use std::convert::TryInto;
 use std::fs;
 use std::path::PathBuf;
 use crate::faas_interface::FaaSFunctionSignature;
+use std::collections::HashSet;
 
 // TODO: remove and use mutex instead
 unsafe impl Send for FluenceFaaS {}
```

Comment:
Do we still need this after corresponding PR in the fluence repo?

Reply:
I was going to check it, but forgot. Thanks!

Reply:
Unfourtunately since it is a part of NetworkBehaviour, it doesn't work. I'll try later to wrap it with mutex, but I don't think it would help since I don't think a can instruct libp2p to lock and unlock the state in any way. But I'm a bit loose on the details, maybe it will work.